docs/rfc/block-registration.md

```diff
@@ -331,6 +331,36 @@ Block type editor style definition. It will only be enqueued in the context of t
 
 Block type frontend style definition. It will be enqueued both in the editor and when viewing the content on the front of the site.
 
+### Provides Context
+
+* Type: `object`
+* Optional
+* Localized: No
+* Property: `providesContext`
```

Comment:
Does this property map to `provides_context` in PHP?

Reply:
> Does this property map to `provides_context` in PHP?

Technically there's no mapping today, but I've mentioned it in the original comment as a potential blocking decision:

>A decision at https://github.com/WordPress/gutenberg/pull/22519#discussion_r431461183 in regards to registering block types using snake-cased `provides_context`, rather than camel-cased `providesContext`.